### DISCUSSION TITLE
Is there a way to add books to multiple shelves?

### DISCUSSION THREAD

**Discussion by spiderkitten** [bookwyrm-social/bookwyrm#2876] [Q&A]
I like to have a shelf for the year I read a book, but also for the type of book it was, so for example a graphic novel I read this year would go on "2023 reads" and "graphic novels"   

also is this the best place for help questions? the documentation is minimal and there's not a lot of help elsewhere (googling).

**Comment by dcchambers** [comment]
You can add a book to as many lists as you want. Simply create the list(s) from the `Lists` page (eg https://bookwyrm.social/list).

Then open the lists you want, search for a book under the `Add Books` section on the left-hand side, and add it to the list. AFAIK you need to open each list and add books from the list itself. There is no way to add a book to a list from the book's page...that would definitely be a nice feature to have in the future.

**Comment by mouse-reeve** [comment]
It is possible for a book to be both on a custom shelf and on an exclusive shelf (to-read, reading, read, or stopped reading), but it's weird and hard to get that to work. This would be fixed by https://github.com/bookwyrm-social/bookwyrm/pull/2170
